src/screens/admin/AddFields.tsx

```
@@ -0,0 +1,475 @@
+    try {
+      const payload: AddFieldPayload = {
+        name: form.name,
+        label: form.label,
+        type: form.type,
+        ordering: fields.length + 1,
+        fieldParams: form.type === 'drop_down' ? { options: form.options } : null,
+        fieldAttributes: { isEditable: form.isEditable, isRequired: form.isRequired },
+      };
+      await addField(payload);
+      toast({
+        title: editingIndex !== null ? 'Field updated' : 'Field added',
```

ESLint

[error] Replace `········` with `↹↹↹↹`

(prettier/prettier)